### ISSUE TITLE
Password Grant validates without  filling the form on swagger

### ISSUE THREAD

**Issue by orthagh** [glpi-project/glpi#18006]
The error returned by `api.php/token` endpoint:

```json
{"status":"ERROR_INVALID_PARAMETER","title":"Invalid OAuth token","detail":"The JWT string must have two dots"}
````

The form although validated, doesn't conclude in a valid state, all other endpoints the same json error

**Comment by cconard96** [comment]
I cannot recreate the issue.

In Swagger UI, providing just login and password for password grant shows an error that there are fields missing.
Providing all info authenticates properly in both Swagger UI and calls using Insomnia.

**Comment by orthagh** [comment]
I don't know, In fact, I just tested without filling anything in the Password Grant form, no error are reported, and I pass on a "logged" state.
I saw the two dots errors in POST response.

I didn't find any errors, in js console, nor in glpi logs.

**Comment by orthagh** [comment]
You may want to test on a fresh codespace I just launched/setup:

https://jubilant-palm-tree-xxq7w67443pj5r-8080.app.github.dev

In this case, I have the error reported (400), but the issue with "The JWT string must have two dots" is present.
Maybe we can succeed in fixing this one

**Comment by cconard96** [comment]
Should have been fixed by #18010

**Comment by orthagh** [comment]
Indeed fixed
